docs/pages/database-access/auto-user-provisioning/mysql.mdx

```diff
@@ -15,6 +15,10 @@ Automatic user provisioning is not compatible with MySQL versions lower than
 8.0.
 </Admonition>
 
+<Admonition type="warning" title="RDS compatibility">
```

Comment:
Having two Admonitions in a row looks pretty busy. I would take the text out of the Admonitions and make them additional bullets in the list above.

Reply:
Thanks Paul! Updated.